docs/Getting_Started_Guide.md

```diff
@@ -0,0 +1,930 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Object Pascal Style Guide](#object-pascal-style-guide)
+- [Preface](#preface)
+- [Introduction](#introduction)
+  - [Background](#background)
+  - [Acknowledgments](#acknowledgments)
+- [Source Files Object Pascal source is divided up primarily into](#source-files-object-pascal-source-is-divided-up-primarily-into)
+  - [Source-File Naming Object Pascal supports long file names. If](#source-file-naming-object-pascal-supports-long-file-names-if)
```

Comment:
doctoc needs to be run against this version of the document, TOC is not in sync.  I'll about installing Node and doctoc later on unless you want to take care of it @kotp.